src/main/webapp/src/app/ranking/ranking-item/ranking-item.component.html

```diff
@@ -1,8 +1,18 @@
-<div>
-  <img height="200" src={{hiddenGem.photo}} alt={{hiddenGem.name}}>
-  <p id="name">{{hiddenGem.name}}</p>
-  <p id="business_type">{{hiddenGem.business_type}}</p>
-  <p id="address">{{hiddenGem.address}}</p>
-  <p id="price_level">{{hiddenGem.price_level}}</p>
-  <p id="rating">{{hiddenGem.rating}}</p>
-</div>
+<mat-card class="mat-elevation-z2">
```

Comment:
Potentially making a more understandable class name e.g. rankingItemCard?. I don't really understand what mat-elevation-z2 means

Reply:
+1 I think starting it with mat implies it comes from the material stylesheet to me. 
also agree elevation is confusing, I assume this is what's giving it drop shadow around the cards? I think giving it a name to do with what changes it's making is good though.

Reply:
Yes, this class comes from the material stylesheet to create the shadow around the card (https://material.angular.io/guide/elevation), if I change it, the "elevation" will be gone. Should I add an id to the mat-card tag to give it an id name? Or is that not necessary?

Reply:
oh I see, you were just modifying this class in your stylesheet, that was what threw me off. 
I think giving the card an id and assigning your custom styling to that id might be a better way to do it, that way if you use the dropshadow elsewhere the padding settings don't accidentally come along for the ride, but not anything that needs to be fixed urgently.